config/routes.rb

```diff
@@ -27,6 +27,11 @@
                  constraints: ->(_) { FeatureFlipper.show_education_benefit_form? }
     end
 
+    resource :health_care_application, only: [:create, :index] do
+      post :create
+      get :index
```

Comment:
Everything @austin-martinez said, you can also limit CRUD operations using `only:` there should be examples already in the routes file on how to do that. 

But I have to agree with @austin-martinez if it is a single resource, and a vet can only ever have one (for example a user can only have 1 profile). Then you should use resource. The name will be singular, but your controller will still be pluralized. More information on this and why controller names are always pluralized, even for singular resources, [here](http://guides.rubyonrails.org/routing.html)